src/main/resources/ttc.properties

```diff
@@ -76,10 +76,9 @@ de.konfidas.ttc.messages.systemlogs.errorDescriptionNotFound = Fehler beim Parse
 de.konfidas.ttc.messages.systemlogs.errorDeviceInformationSetEndedEarly = DeviceInformationSet in updateTime endete vorzeitig.
 de.konfidas.ttc.messages.systemlogs.errorDeviceInformationSetOfUpdateTimeEndedEarly = DeviceInformationSet in updateTime endete vorzeitig.
 de.konfidas.ttc.messages.systemlogs.errorEarlyEndOfSystemOperationData = Fehler beim Parsen des systemOperationDataContent. Vorzeitiges Ende von systemOperationData.
-de.konfidas.ttc.messages.systemlogs.errorLoggingParameterNotOfTypeASN1Sequence = Fehler beim Parsen des loggingParameter. Der Eintrag sollte vom Typ ASNSequence sein.
+de.konfidas.ttc.messages.systemlogs.errorParametersDoesNotStartWithOctetString = Fehler beim Parsen des loggingParameter. Der Eintrag sollte vom Typ ASN1OctetString sein.
```

Comment:
-> `Fehler beim Parsen des loggingParameter. Das Pflichtfeld loggingParameter startet nicht mit einem Octet String.` for consitency